templates/terraform/examples/iap_identity_aware_proxy_client.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_project" "project" {
+  project_id = "tf-test%{random_suffix}"
+  name       = "tf-test%{random_suffix}"
+  org_id     = "%{org_id}"
+}
+
+resource "google_project_service" "project_service" {
+  project = google_project.project.project_id
+  service = "iap.googleapis.com"
+}
+
+resource "google_iap_brand" "project_brand" {
+  support_email     = "support@example.com"
```

Comment:
This acceptance test won't succeed unless the specified group is owned by the service account running the test.